Example/ViewController.swift

```diff
@@ -408,6 +411,13 @@ class ViewController: UIViewController {
         mapView.showsUserLocation = true
         mapView.userTrackingMode = .follow
         mapView.showsUserHeadingIndicator = true
+        
+        // Use map style which allows to show road traffic.
+        mapView.styleURL = MGLStyle.navigationDayStyleURL
+        
+        // Override traffic congestion level from `.unknown` to `.low`
+        // for `MapboxStreetsRoadClass` passed in `trafficOverrideStreetsRoadClasses`.
+        mapView.trafficOverrideStreetsRoadClasses = [.motorway]
```

Comment:
Would it be confusing to include this customization by default in the example application? It would mask default behavior that we would otherwise experience during development.

Reply:
I guess you're right. To prevent any shift in results due to usage of this property I'll remove this part for sample app.